src/main/java/com/deavensoft/paketomat/dispatcher/DispatcherServiceImpl.java

```diff
@@ -25,6 +26,7 @@
 @Slf4j
 public class DispatcherServiceImpl implements DispatcherService {
 
+    @Autowired
```

Comment:
- [ ] Delete annotation

Reply:
If I delete that annotation, when I want to get all the dispatchers, I get an error that reads:
_Cannot invoke "com.deavensoft.paketomat.dispatcher.DispatcherRepository.findAll()" because "this.dispatcherRepository" is null_

Reply:
Use @AllArgsConstructor  to generate a constructor for all of your class's fields and use @RequiredArgsConstructor to generate a constructor for all class's fields that are marked as final.

You have two options: 
1. `private **final** DispatcherRepository dispatcherRepository;`
2. Change constructor annotation to  @AllArgsConstructor

Reply:
Done

Reply:
Now, try to delete the annotation 😄